samples/near-realtime-clients/Core_7/ClientHub/StockTickHandler.cs

```diff
@@ -1,20 +1,28 @@
 ﻿using System.Threading.Tasks;
-using Microsoft.AspNet.SignalR;
+using Microsoft.AspNetCore.SignalR;
 using NServiceBus;
 using NServiceBus.Logging;
 
+namespace ClientHub;
+
+
 #region StockTickHandler
 
 public class StockTickHandler :
     IHandleMessages<StockTick>
 {
-    static ILog log = LogManager.GetLogger<StockTickHandler>();
-    IHubContext hub = GlobalHost.ConnectionManager.GetHubContext<StockTicksHub>();
+    static readonly ILog log = LogManager.GetLogger<StockTickHandler>();
+    readonly IHubContext<StockTicksHub> hub;
+
+    public StockTickHandler(IHubContext<StockTicksHub> hub)
+    {
+        this.hub = hub;
+    }
 
     public Task Handle(StockTick message, IMessageHandlerContext context)
     {
         log.Info($"StockTick event received for Symbol {message.Symbol} with timestamp: {message.Timestamp:O}. Press any key to exit.");
-        return hub.Clients.All.PushStockTick(message);
+        return hub.Clients.All.SendAsync("PushStockTick", message);
```

Comment:
Shouldn't this be in its own handler where Handle would first publish a StockUpdated event? It would show that you could have a "SignalR" channel handler but optionally also others.